execution-environment.yml

```diff
@@ -2,8 +2,13 @@
 version: 3
 images:
   base_image:
-    name: quay.io/centos/centos:stream9
+    name: quay.io/centos/centos:stream9-minimal
```

Comment:
any particular reason for switching from stream9 to stream9-minimal?

Reply:
The `stream9-minimal` images do not include python at all out of the box, which allows us to avoid unnecessarily bundling two versions of python in awx-ee (3.9 + 3.11). That was the primary motivation.